test/unit/expiration_process_test.lua

```diff
@@ -241,3 +241,41 @@ function g.test_default_tuple_drop_function(cg)
         t.assert_equals(space:count{}, 0)
     end)
 end
+
+function g.test_tuples_per_iteration(cg)
+    local space = cg.space
+    local space_archive = cg.space_archive
+    local task_name = cg.task_name
+
+    local total = 10
+    local time = fiber.time()
+    for i = 1, total do
+        space:insert({i, tostring(i), time})
+    end
+    t.assert_equals(space:count{}, total)
+
+    cg.task = expirationd.start(task_name, space.id, check_tuple_expire_by_timestamp,
+        {
+            process_expired_tuple = put_tuple_to_archive,
+            args = {
+                field_no = 3,
+                archive_space_id = space_archive.id
+            },
+            iteration_delay = 1,
+            vinyl_assumed_space_len = 5, -- iteration_delay will be 1 sec
```

Comment:
Let's comment on one style.
```suggestion
            vinyl_assumed_space_len = 5, -- Iteration_delay will be 1 sec.
```